syntax = "proto3";

package autoxtime.proto;

message Driver {
  optional int64 driver_id = 1;
  // primary key 3
  optional string first_name = 2;
  optional string last_name = 3;
  // primary key 4
  optional string email = 4;
  // primary key 5
  optional string phone_number = 5;
  // primary key 2
  optional int32 msr_id = 6;
  // primary key 1
  optional int32 scca_id = 7;
  optional int32 work_req = 8;
  optional int32 work_skill = 9;
}
